doc/deployment/import-kubernetes-context.md

```diff
@@ -0,0 +1,114 @@
+# Import a Kubernetes Context
+
+**Note:** The steps in this section apply to your configuration only
+if you deployed Pachyderm from a manifest created by the `pachctl deploy`
+with the `--dry-run` flag. If you did not use the `--dry-run` flag, skip this
+section.
+
+When you run the `pachctl deploy` command with `--dry-run` flag for any
+underlying infrastructure, such as GKE, Amazon, local, or any other, you do
+not create an actual Pachyderm cluster, Instead you create a Kubernetes
+deployment manifest that you can later use to deploy a Pachyderm cluster.
+
+You can use that manifest with a standard `kubectl apply` command to deploy
+Pachyderm. For example, if you have created a manifest called
+`test-manifest.yaml`, you can deploy a Pachyderm cluster by running the
+following command:
+
+```bash
+kubectl apply -f test-manifest.yaml
+```
+
+Typically, when you run `pachctl deploy`,
+Pachyderm creates a new Pachyderm context with the information from
+the current
+[Kubernetes context](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context)
+embedded into it.
+
+When you use the `--dry-run` flag, the Pachyderm context is not created.
+Therefore, if you deploy a Pachyderm cluster from a manifest that you have
+created earlier, you need to manually create a new Pachyderm context with
+the embedded current Kubernetes context and activate that context.
+
+To import a Kubernetes context, complete the following steps:
+
+1. Deploy a Pachyderm cluster from the Kubernetes manifest that you have
+   created when you ran the `pachctl deploy` command with the `--dry-run`
+   flag:
+
+   ```bash
+   $ kubectl apply -f <manifest.yaml>
+   clusterrole.rbac.authorization.k8s.io/pachyderm configured
+   clusterrolebinding.rbac.authorization.k8s.io/pachyderm configured
+   deployment.apps/etcd configured
+   service/etcd configured
+   service/pachd configured
+   deployment.apps/pachd configured
+   service/dash configured
+   deployment.apps/dash configured
+   secret/pachyderm-storage-secret configured
+   ```
+
+1. Verify that the cluster was successfully deployed:
+
+   ```bash
+   $ kubectl get pods
+   NAME                     READY   STATUS    RESTARTS   AGE
+   dash-64c868cc8b-j79d6    2/2     Running   0          20h
+   etcd-6865455568-tm5tf    1/1     Running   0          20h
+   pachd-6464d985c7-dqgzg   1/1     Running   0          70s
+   ```
+
+   You must see all the `dash`, `etcd`, and `pachd` pods running.
+
+1. Create a new Pachyderm context with the embedded Kubernetes context:
+
+   ```bash
+   $ pachctl config set context <new-pachyderm-context> -k `kubectl config current-context`
+   ```
+
+   **Example:**
+
+   ```bash
+   $ pachctl config set context test-context -k `kubectl config current-context`
```

Comment:
I dont think you need this example and the above. 

pachctl config set context <new-pachyderm-context> -k `kubectl config current-context` is pretty c;ear without needing to show an example where you replace one arugment

Reply:
I can remove them, but overall they don't hurt, IMO.